src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -568,6 +570,46 @@ public static List<ClusterMetadata> tcmLoadRange(long min, long max)
         return afterLoad;
     }
 
+    /**
+     * This method exists due to the fact that we define a retry policy for TCM to follow, and then TCM ignores it and does no retries...
+     */
+    private static List<ClusterMetadata> reconstruct(long min, long max)
+    {
+        Epoch start = Epoch.create(min);
+        Epoch end = Epoch.create(max);
+        Retry.Deadline retryPolicyThatGetsIgnored = Retry.Deadline.retryIndefinitely(DatabaseDescriptor.getCmsAwaitTimeout().to(NANOSECONDS),
```

Comment:
I think we need to make `AbstractLocalProcessor#getLogState` (currently as passthrough) retry. Or have two each implementation retry individually.

Reply:
im trying out a refector to push retries into TCM

Reply:
pushed retry into TCM while respecting the semantics I want